src/h5web/vis-packs/core/heatmap/HeatmapVis.tsx

```diff
@@ -40,50 +40,47 @@ function HeatmapVis(props: Props) {
     ordinateParams = {},
   } = props;
 
+  const { label: abscissaLabel, value: abscissaValue } = abscissaParams;
+  const { label: ordinateLabel, value: ordinateValue } = ordinateParams;
+
   const { rows, cols } = getDims(dataArray);
   const aspectRatio = keepAspectRatio ? cols / rows : undefined; // width / height <=> cols / rows
 
-  const abscissas = getPixelEdges(abscissaParams.value, cols);
-  const abscissaToIndex = getValueToIndexScale(abscissas);
-  const abscissaDomain = useMemo(() => getDomain(abscissas), [abscissas]);
+  const abscissas = usePixelEdges(abscissaValue, cols);
```

Comment:
Looking back at this, I find `getPixelEdges` not clear enough... 

What about you ? Do you think `getPixelEdgesCoordinates` would be better ?

Reply:
Actually, how about `useIndices`? I find the term "coordinates" inaccurate.

Reply:
Pushed. Let me know what you think.

Reply:
Well if `abscissaValue` is defined, these are not indices...